.github/react-workflow-template.yaml

```
@@ -77,3 +77,46 @@ jobs:
           docker build . -t arkivverket.azurecr.io/da-mottak/!COMPONENT_NAME!:${{ github.sha }}
           docker push arkivverket.azurecr.io/da-mottak/!COMPONENT_NAME!:${{ github.sha }}
 
+  update-deployment-version:
+    needs:
+      - build-and-push-docker-image
+    if: github.ref == 'refs/heads/develop'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+
+      - name: Install Python and Poetry
+        run: |
+          python -m pip install -U pip
+          pip install poetry
+
+      - uses: actions/checkout@v2
+        with:
+          repository: arkivverket/mottak-k8s-config
+          ref: master
+          ssh-key: ${{ secrets.MOTTAK_K8S_CONFIG_SECRET }}
+
+      - name: Install dependencies
+        run: |
+          poetry install
```

```suggestion
          poetry install
          
```

---

Trailing spaces tenker du?
Da burde det heller være
```suggestion
          poetry install

```

Slik at det ikke blir en tom linje med masse spaces

---

Godt poeng, jeg slurvet her